tracecat/records/router.py

```diff
@@ -0,0 +1,175 @@
+import uuid
+from typing import Annotated
+
+from fastapi import APIRouter, HTTPException, Query, status
+
+from tracecat.auth.credentials import RoleACL
+from tracecat.db.dependencies import AsyncDBSession
+from tracecat.entities.router import router as entities_router
+from tracecat.entities.service import EntityService
+from tracecat.records.model import RecordCreate, RecordRead, RecordUpdate
+from tracecat.records.service import RecordService
+from tracecat.types.auth import Role
+from tracecat.types.exceptions import TracecatNotFoundError
+from tracecat.types.pagination import (
+    CursorPaginatedResponse,
+    CursorPaginationParams,
+)
+
+records_router = APIRouter(prefix="/records", tags=["records"])
+
+WorkspaceUser = Annotated[
+    Role,
+    RoleACL(
+        allow_user=True,
+        allow_service=False,
+        require_workspace="yes",
+    ),
+]
+
+
+@entities_router.get(
```

Comment:
can this go in tracecat/entities/router.py? then you can rename `records_router` -> `router`